x-pack/plugins/canvas/canvas_plugin_src/functions/server/src/escount.js

```diff
@@ -4,25 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
 import { buildESRequest } from '../../../../server/lib/build_es_request';
 
 export const escount = () => ({
   name: 'escount',
   type: 'number',
-  help: 'Query elasticsearch for a count of the number of hits matching a query',
+  help: i18n.translate('xpack.canvas.functions.escountHelpText', {
```

Comment:
please add .escount. namespace for all ids in this file. Do the same for other files